### PR TITLE
22 functional requirements

### PR DESCRIPTION
**Pull request by mkuelker**
Oh, great council of the speaking portal project!

Approve my PR before I perish and die.

### CODE REVIEW

**Review by LoganParker — CHANGES_REQUESTED** [review]
Confused as to why the tech stack content is in here, maybe remove those changes from this PR and then if there's any suggestions just comment em on #61 
- Looks like maybe those are my commits somehow, so as long as you're ok with it I'll delete the file changes on this PR 


Functional requirements look good, one thing I'd mention is to make sure we're observing the not so strict character limit of 120 per line in markdown files, it makes these reviews a lot easier (and you get more lines committed)

**Comment by mkuelker** [comment]
> Confused as to why the tech stack content is in here, maybe remove those changes from this PR and then if there's any suggestions just comment em on #61
> 
> * Looks like maybe those are my commits somehow, so as long as you're ok with it I'll delete the file changes on this PR
> 
> Functional requirements look good, one thing I'd mention is to make sure we're observing the not so strict character limit of 120 per line in markdown files, it makes these reviews a lot easier (and you get more lines committed)

I based the branch on the 23_requirements_report branch. I think you had your tech stack on there, hence the mixup

**Review by mkuelker — COMMENTED** [review]
Deleted Slide and tech slides. This should be ready to merge now

**Review by LoganParker — APPROVED** [review]
Thanks for making those changes Matt! I'd check out Edouard's comment to ensure that we have the full scope of functional requirements done, but otherwise looks great!

**Review by edouarde1 — CHANGES_REQUESTED** [review]
Great job Matt! I have one suggestion about the func requirements

File: docs/reports/requirements/3_func-req.md [hunk 1]
```diff
@@ -9,3 +11,8 @@ Feature would preferably support some form of facial expression
 Any voice selected must be able to be synced to feature animation.
 Any language selected must be able to be synced to the feature animation
 
+# PDF CONTENT
+
+In our conversations with the client, the following functional requirements were identified: 
+Firstly, our users should be able to interact with the feature through a UI. Kukarella has already supplied all of the front-end UI requirements, however for testing our team will use a mock UI. This will assist with user testing and feedback while our codebase is still separate from Kukarella development. Early testing and examples from other groups proved that lip-syncing alone is not enough to produce a convincing animation—and as such facial expressions are required.  The client also noted that any voice or language selected should be able to seamlessly integrate with our animation output. This, along with user text and the generated voice would be required for this feature to work.
```
[the remark this answers is not in the document]
Comment (mkuelker) [reply docs/reports/requirements/3_func-req.md:17]: I've included the first bullet point in the paragraph but the text is a little unclear, I'll add better language to make it more transparent. 
For the second bullet point, I included it as a non-functional requirement, as technically the feature could work without being made into an API. (it just wouldn't be any good.) 
I'll add the animation as an mp4 in there.

**Pull request merged**